main.go

```diff
@@ -68,6 +68,7 @@ func main() {
 		clients  = flag.Int("clients", 10, "Number of clients to start")
 		format   = flag.String("format", "text", "Output format: text|json")
 		quiet    = flag.Bool("quiet", false, "Suppress logs while running")
+		numTopics = flag.Int("numTopics", 1, "Number of different topics")
```

Comment:
`topics` will be consistent with the naming of other parameters? (sorry missed at first look)